assignment.js

```diff
@@ -66,6 +86,19 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const numberList = [13, 4, 57, 38];
```

Comment:
@jaygo90  nice here of using an `if` statement to keep track of the largest number. I used an alternative approach (that violated the requirements) of using `sort()` to solve this. You can check out the approach [here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L86)